spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe GamesController, type: :controller do
+  describe "games#create action" do
+    it "should successfully create new game in the database" do
+      post :create, params: { game: {game_id: 1, user_white_id: 1, user_black_id: 2} }
+      expect(response).to redirect_to game_path
```

Comment:
I'd also add in a couple tests to verify the correct attributes are being set.

Such as

`expect(Game.last.user_white_id).to eq 1`